.buildkite/pipelines/sync_pipeline.yml

```
@@ -9,7 +9,20 @@ steps:
     depends_on: "start"
     commands:
       - cd .metrics/buildkite
+      - DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get install -y curl 
       - curl -o ci_integrations -k https://github.com/platform-platform/monorepo/releases/download/ci_integrations-snapshot/ci_integrations_linux -L
       - chmod a+x ci_integrations
       - eval "echo \"$(sed 's/"/\\"/g' integration_config.yml)\"" >> integration.yml
       - ./ci_integrations sync --config-file integration.yml
```

Would it be better as we don't generate coverage in the dummy pipeline? So the `ci_integrations` won't try to fetch appropriate artifacts.
```suggestion
      - ./ci_integrations sync --config-file integration.yml --no-coverage
```